Fabric.Authorization.AccessControl/src/app/modules/access-control/custom-group/custom-group.component.ts

```diff
@@ -515,9 +515,19 @@ export class CustomGroupComponent implements OnInit, OnDestroy {
       takeUntil(this.ngUnsubscribe))
       .subscribe(null, (error) => {
         if (error.statusCode === 409) {
-          this.groupNameInvalid = true;
-          this.groupNameError = `Could not create group named "${this.groupName}". ` +
-          `A group with the same name exists as a Custom group or a Directory group`;
+          if (error.message.startsWith("The associated user or group name should not be the same as an existing custom group:"))
```

Comment:
Can we just use the `groupNameInvalid` and `groupNameError` fields for both cases and get rid of the if/else? What is `error.message` set to in the `else` case where we're hard-coding our own error message? Can we just always take the `error.message` that comes back in the API response?

Reply:
It shows up in the wrong spot, need to use the one I created for the initial weirdness bug.

Reply:
I'm just more concerned the `if` statement uses message text returned from the API, so if that changes, then this has to change.

Reply:
Should I throw a different error instead of 409, so that text won't be an issue?

Reply:
Not sure we should change the HTTP response if 409 is the appropriate one for this scenario...